crates/harp/src/command.rs

```diff
@@ -22,11 +22,20 @@ pub fn r_command<F>(build: F) -> io::Result<Output>
 where
     F: Fn(&mut Command),
 {
-    assert!(COMMAND_R_LOCATIONS.len() > 0);
```

Comment:
Split into `r_command()` and `r_command_from_path()` or `r_home_from_path()` depending on what makes sense